### ISSUE TITLE
[1.12.2] Reactor fuel vanishing without cause

### ISSUE THREAD

**Issue by Dusty-Sprinkles** [ZeroNoRyouki/BigReactors#176]
I have been playing on a server running MC 1.12.2 for a while now, and I recently built a 6x6x6 (internal volume) reactor. This reactor had been running fine until a period where it was offline, and all of the fuel was drained without leaving any byproduct. (while at 100% radioactivity, and 20 C)

**Comment by ZeroNoRyouki** [comment]
Which version of ER are you using? Are you playing a public modpack (which one?) or a custom one?

**Comment by Dusty-Sprinkles** [comment]
Version 0.4.5.48, The modpack is Kingdom of Delerior, and this has happened in previous versions of the pack to another person

**Comment by ZeroNoRyouki** [comment]
Are you sure that the fuel was not consumed while the reactor was still "hot" after you turned it off?

**Comment by Dusty-Sprinkles** [comment]
Yes, I stay by my reactor to make sure it drops down to non-functioning temperature and radioactivity.

**Comment by NanobotZ** [comment]
I've noticed that there is a bug when the whole reactor is not fit into only one chunk - some chunk load error when the whole reactor is not loaded at once

**Comment by xxxLEOPARDxxx** [comment]
I confirm!
Utal already struggle with this problem!
Sometimes the reactor itself turns off and all the fuel disappears.
